### ISSUE TITLE
Asset Management - Report Creation

### ISSUE THREAD

**Issue by domenicocodispoti** [grokability/snipe-it#10525]
### Snipe-IT Version

5.3.7

### Operating System

Ubutnu 21.04

### Web Server

Mariadb 10.6.5

### PHP Version

7.4.1

### Is your feature request related to a problem? Please describe. A clear and concise description of what the problem is. Ex. I'm always frustrated when [...]

An import report is missing in the Asset Management software.

### Describe the solution you'd like A clear and concise description of what you want to happen.

Basically I need to create a new report about accessories which indicate who that kind of accessorie is assigned to.

**Comment by King-GO** [comment]
[Check here](https://github.com/snipe/snipe-it/discussions/8963#discussioncomment-354764)
